app/models/news_article.rb

```diff
@@ -8,6 +8,8 @@ class NewsArticle < Newsesque
   validates :news_article_type_id, presence: true
   validate :non_english_primary_locale_only_for_world_news_story
 
+  validate :policies_are_not_associated, unless: :can_be_related_to_policies?
```

Comment:
Would `validates :edition_policies, absence: true, if: :world_news_story?` work as well? That's why I'm using in my (as of yet unmerged) [PR](https://github.com/alphagov/whitehall/pull/3225)